src/export/node_export_algorithm.rb

```diff
@@ -24,6 +24,11 @@ def run
     edges = Graph.instance.edges.values
 
     static_groups = StaticGroupAnalysis.find_static_groups
+
+    rotary_hinges_ids = check_for_only_simple_hinges static_groups
+
+    offset_rotary_hinge_hubs(rotary_hinges_ids, static_groups)
```

Comment:
I don't think there are ids stored in `rotary_hinges_ids` but arrays of nodes, right?